### PR TITLE
Scala 2.13.11 draft release notes

### PR DESCRIPTION
**Pull request by SethTisue**
feedback welcome

### CODE REVIEW

File: releases/2.13.11.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+The following changes are highlights of this release:
+
+### Collections
+
+* Faster `Vector` concatenation ([#10159](https://github.com/scala/scala/pull/10159) by [@ansvonwa](https://github.com/ansvonwa))
+* New `LinkedHashMap`/`LinkedHashSet` implementation ([#10221](https://github.com/scala/scala/pull/10221) by [@liang3zy22](https://github.com/liang3zy22))
+
+### Compatibility
+
+* Support JDK 21 by upgrading to ASM 9.5 ([#10363](https://github.com/scala/scala/pull/10363), [#10184](https://github.com/scala/scala/pull/10184) by [@Philippus](https://github.com/Philippus))
```
Comment (Philippus) [inline releases/2.13.11.md]:
```suggestion
* Support JDK 20 and 21 by upgrading to ASM 9.5 ([#10363](https://github.com/scala/scala/pull/10363), [#10184](https://github.com/scala/scala/pull/10184) by [@Philippus](https://github.com/Philippus))
```

File: releases/2.13.11.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+As usual for our minor releases, Scala 2.13.11 is [binary-compatible](https://docs.scala-lang.org/overviews/core/binary-compatibility-of-scala-releases.html) with the whole Scala 2.13 series.
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+We especially acknowledge and thank A. P. Marki, also known as Som Snytt, who is responsible for an large share of the improvements in this release.
```
Comment (Philippus) [inline releases/2.13.11.md]:
```suggestion
We especially acknowledge and thank A. P. Marki, also known as Som Snytt, who is responsible for a large share of the improvements in this release.
```

**Comment by michelou** [comment]
@SethTisue Missing closing parenthesis at the end of the line : 

* Support Java 17 `sealed` in Java sources and classfiles ([#10348](https://github.com/scala/scala/pull/10348), [#10105](https://github.com/scala/scala/pull/10105) by [@som-snytt](https://github.com/som-snytt)

**Comment by SethTisue** [comment]
@michelou fixed, thanks. @Philippus thanks for your fixes as well

**Comment by som-snytt** [comment]
@michelou thanks, I was almost left dangling.

File: releases/2.13.11.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* Make scalac's argument-file processing more like javac's in handling spaces and line breaks ([#10319](https://github.com/scala/scala/pull/10319) by [@som-snytt](https://github.com/som-snytt))
+* Bump JLine and JNA versions ([#10318](https://github.com/scala/scala/pull/10318))
+* javabootclasspath supersedes jrt ([#10336](https://github.com/scala/scala/pull/10336) by [@som-snytt](https://github.com/som-snytt))
+* Report total errors on stderr ([#10272](https://github.com/scala/scala/pull/10272) by [@som-snytt](https://github.com/som-snytt))
+* Fix pathologically slow compilation of some pattern matches since 2.13.7 ([#10258](https://github.com/scala/scala/pull/10258) by [@KisaragiEffective](https://github.com/KisaragiEffective))
+* Provide `equals` and `hashCode` on `IterableWrapper` again (fixing 2.13.9 regression) ([#10205](https://github.com/scala/scala/pull/10205) by [@som-snytt](https://github.com/som-snytt))
+* Improvements to `-Vimplicits`; errors now show complete implicit search tree ([#9944](https://github.com/scala/scala/pull/9944) by [@tribbloid](https://github.com/tribbloid))
```
Comment (som-snytt) [inline releases/2.13.11.md:47]: * `-Vprint` defaults to `-Vprint:typer`. Phases may be prefixed with `~` to mean "before and after"; for example, `-Vprint:~tailcalls` prints trees before tailcalls (at the end of fields phase) and after.  ([#10173](https://github.com/scala/scala/pull/10173) by [@som-snytt](https://github.com/som-snytt))

Comment (SethTisue) [reply releases/2.13.11.md:47]: added, and I added the "release notes" label to the PR in question.

File: releases/2.13.11.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+* Deprecate infix type args, as they are dropped in Scala 3 ([#10255](https://github.com/scala/scala/pull/10255) by [@som-snytt](https://github.com/som-snytt))
+* Allow eta-expansion of methods with dependent types ([#10166](https://github.com/scala/scala/pull/10166))
+* Abstract parent `copy` does not suspend case `copy` ([#10085](https://github.com/scala/scala/pull/10085) by [@som-snytt](https://github.com/som-snytt))
+* Don't GLB binders of type patterns, use the type directly ([#10247](https://github.com/scala/scala/pull/10247))
+* Under `-Xsource:3`, adjust 2.13.9 change to ignore override type for whitebox macro expansion ([#10188](https://github.com/scala/scala/pull/10188) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, use narrowest type for pt of override  ([#10198](https://github.com/scala/scala/pull/10198) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, warn about change to parenless lambda parens ([#10320](https://github.com/scala/scala/pull/10320) by [@som-snytt](https://github.com/som-snytt))
```
Comment (som-snytt) [inline releases/2.13.11.md:29]: Could add a section just for `-Xsource:3`

### Migration to Scala 3 under `-Xsource:3`

or

### `-Xsource:3`: Mutant Scala

File: releases/2.13.11.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+### Compatibility
+
+* Support JDK 20 and 21 by upgrading to ASM 9.5 ([#10363](https://github.com/scala/scala/pull/10363), [#10184](https://github.com/scala/scala/pull/10184) by [@Philippus](https://github.com/Philippus))
+* Fix crash reading JDK 21 classfiles ([#10397](https://github.com/scala/scala/pull/10397))
+* Support Java 17 `sealed` in Java sources and classfiles ([#10348](https://github.com/scala/scala/pull/10348), [#10105](https://github.com/scala/scala/pull/10105) by [@som-snytt](https://github.com/som-snytt))
+
+### Align with Scala 3
+
+* Accept `implicit _`, `implicit (x: Int)` in lambdas, like Scala 3 does ([#10327](https://github.com/scala/scala/pull/10327) by [@som-snytt](https://github.com/som-snytt))
+* Inherited members no longer take precedence over outer definitions ([#10220](https://github.com/scala/scala/pull/10220), [#10339](https://github.com/scala/scala/pull/10339))
```
Comment (som-snytt) [inline releases/2.13.11.md]: Under -Xsource:3, warn that inherited members no longer take precedence over outer definitions in Scala 3

File: releases/2.13.11.md (hunk 4 above)
Comment (SethTisue) [reply releases/2.13.11.md:29]: meh... no strong opinion either way, but I think I like it a bit better as-is

File: releases/2.13.11.md [hunk 6]
```diff
@@ -51,6 +51,15 @@ The following changes are highlights of this release:
 
 For the complete 2.13.11 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.11) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.11).
 
+## Known issues
+
+A few regressions have already cropped up:
+
+* Exhaustivity checker emits spurious warning when matching on Java enum type (2.13.11 regression) ([scala/bug#12800](https://github.com/scala/bug/issues/12800))
+* Duplicated `@Deprecated` annotations when implementing deprecated Java interface cause `java.lang.annotation.AnnotationFormatError` when accessed via Java reflection (2.13.11 regression) ([scala/bug#12799](https://github.com/scala/bug/issues/12799))
```
Comment (som-snytt) [inline releases/2.13.11.md]: `when extending a Java interface with a deprecated default method`. As my Grammie used to say, Cut 'em thin, sure to win.

File: releases/2.13.11.md [hunk 7]
```diff
@@ -51,6 +51,15 @@ The following changes are highlights of this release:
 
 For the complete 2.13.11 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.11) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.11).
 
+## Known issues
+
+A few regressions have already cropped up:
```
Comment (som-snytt) [inline releases/2.13.11.md]: "A few" -> "A murder of"
"cropped up" -> "come surging over the levee" or today's metaphor is surging down the Dnipro

File: releases/2.13.11.md (hunk 6 above)
Comment (SethTisue) [reply releases/2.13.11.md]: thanks. the lengthy/detailed description helps convey the obscurity of the corner case

File: releases/2.13.11.md (hunk 7 above)
Comment (SethTisue) [reply releases/2.13.11.md]: Must. Resist. Urge. To. Wax. Poetic.

**Comment by SethTisue** [comment]
notes are at https://github.com/scala/scala/releases/tag/v2.13.11 now

**Pull request closed without merging**